plugins/modules/ec2_ami.py

```diff
@@ -116,9 +116,15 @@
   launch_permissions:
     description:
       - Users and groups that should be able to launch the AMI.
-      - Expects dictionary with a key of C(user_ids) and/or C(group_names).
+      - Expects dictionary with key of any of following
+          - C(user_ids)
+          - C(group_names)
+          - C(org_arns)
+          - C(org_unit_arns)
```

Comment:
This feels like it would be better handled as suboptions, so we can set the types on the suboptions?

Reply:
added the suboptions for `launch_permissions`.
https://github.com/ansible-collections/amazon.aws/pull/1690/commits/48036120be15c8c1d937d6bcef107294c1d1eb4f